client/components/Navigation.js

```diff
@@ -50,24 +42,19 @@ export default class Navigation extends React.PureComponent {
     return (
       <nav className="c_navigation">
         <div className="c_navigation__links">
-          <NavLink
-            path="/"
-          >
-            <FormattedMessage
-              id="navigation.links.home"
-              defaultMessage="Home"
-              description="'Home' link in the navigation bar"
-            />
-          </NavLink>
-          <NavLink
-            path="/contact"
-          >
-            <FormattedMessage
-              id="navigation.links.contact"
-              defaultMessage="Contact"
-              description="'Contact' link in the navigation bar"
-            />
-          </NavLink>
+          <div className="c_navigation__logo">
+            <Link
+              to="/"
+            >
+              <div style={{
```

Comment:
why inline style here instead of css class?

Reply:
Only because this is meant as a temporary placeholder, and the styles won't apply to the logos once they're added. We could add a class and remove the styles from the .scss when we get the assets, but I figured it's easier this way.